app/utils/discogs.ts

```diff
@@ -1,25 +1,33 @@
 import { SearchResponseSchema } from "./discogs.schemas";
 
-const discogs = {
-  search: async () => {
+type props = string;
+type searchResult = {
+  results: [{ year?: string }];
+};
+
+const token = process.env.DISCOGS_API_KEY!
+console.log("🚀 ~ file: discogs.ts:9 ~ token:", token)
+
 
-    // define url for GET API call
+async function discogs(search: props): Promise<searchResult>  {
+
+  // define url for GET API call
     const searchParams = new URLSearchParams({
-      q: "Nirvana",
+      q: search,
       type: "release",
-      token: process.env.DISCOGS_API_KEY!,
+      token: token,
```

Comment:
you can just inline this also, no need to seperate it out

```suggestion
      token: process.env.DISCOGS_API_KEY!,
```